include/fluent-bit/flb_aws_util.h

```diff
@@ -177,5 +177,17 @@ int flb_read_file(const char *path, char **out_buf, size_t *out_size);
 flb_sds_t flb_get_s3_key(const char *format, time_t time, const char *tag,
                          char *tag_delimiter, uint64_t seq_index);
 
+/*
+ * Count the substring number in string.
+ */
+int count_substring(char *str1, char *str2);
+
+/*
+ * Adjust the time_key output format according to time_key_format.
```

Comment:
Let's make the contract clear in the description. This function sets the out_buf to the time string according to the time_key_format right?

Also, is there truly a concept of keys in this function? Or is just formatting time strings and could be used in other contexts...

Reply:
Can you please update the description according to above?